governance/xc_admin/packages/xc_admin_frontend/next.config.js

```diff
@@ -5,6 +5,7 @@ const nextConfig = {
     externalDir: true,
   },
   webpack(config) {
+    config.resolve.fallback = { fs: false }
```

Comment:
How does this work?

Reply:
this is required for calling fs in next.js